test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -107,6 +108,9 @@ func (s *snapshottableTestSuite) DefineTests(driver TestDriver, pattern testpatt
 		if !ok {
 			e2eskipper.Skipf("Driver %q does not support dynamic provisioning - skipping", driver.GetDriverInfo().Name)
 		}
+		if pattern.VolMode == v1.PersistentVolumeBlock && !dInfo.Capabilities[CapBlock] {
```

Comment:
@jingxu97 brought up a good point that we should move all these driver + testpattern checks to base.go so that every testsuite doesn't have to copy the same logic

Reply:
Done. I also had a cleanup for this driver+testpattern check. I went through all the testsuites and make sure they are all in good position now.